ext/readline/config.m4

```diff
@@ -75,6 +75,11 @@ if test "$PHP_READLINE" && test "$PHP_READLINE" != "no"; then
     -L$READLINE_DIR/$PHP_LIBDIR $PHP_READLINE_LIBS
   ])
 
+  AC_CHECK_DECL([rl_erase_empty_line],
+    [AC_DEFINE([HAVE_ERASE_EMPTY_LINE], [1],
+      [Define to 1 if readline library has rl_erase_empty_line variable.])],,
```

Comment:
why there are two more spaces in indentation?

Reply:
> why there are two more spaces in indentation?

Yes, it seems very strange indeed :D Autoconf's CS is not very sane. This is because it's the 3rd argument of the AC_DEFINE macro.

![image](https://github.com/php/php-src/assets/1614009/f1453139-e58a-4aa3-8284-2e051cc078fb)

This might be perhaps slightly more understandable from other languages:
![image](https://github.com/php/php-src/assets/1614009/6af45b03-af5a-425b-afc0-39093d4cbf88)

It's like CS in other languages, once you get used to it, it somehow feels logical. A bit.

And those quotes `[]` take care that there are as little newlines entered into the generated configure script. Something like this.

Reply:
@petk I see. Many thanks 👍🏻